File: Tarea8_Python/libro.py
import random
class Libro:
    __id = 0
    __name = ""
    __autor = ""
    __disponible = False

    def __init__(self, name, autor):
        self.__id = random.randint(1,1000)
        self.__name = name 
        self.__autor = autor 
        self.__disponible = True

    def get__disponible (self):
        return self.__disponible
    
    def get_autor (self):
        return self.__autor
    
    def get_id(self) :
        return self.__id  
    
    def get_name (self):
        return self.__name
    
    def set__disponible (self, disponible):
         self.__disponible = disponible
    
    
    
    

File: Tarea_8/src/tarea_8/Libro.java

package tarea_8;
import java.util.*;
public class Libro {
    private int id;
    private String nombre;
    private String autor;
    private boolean disponible;

    public Libro(String nombre, String autor) {
        Random rand = new Random();
        this.id = rand.nextInt(1000) + 1;
        this.nombre = nombre;
        this.autor = autor;
        this.disponible = true;
    }

    public int getId() {
        return id;
    }

    public String getNombre() {
        return nombre;
    }

    public String getAutor() {
        return autor;
    }

    public boolean isDisponible() {
        return disponible;
    }

    public void setDisponible(boolean disponible) {
        this.disponible = disponible;
    }
}
